cmd/connector/validate/validate.go

```diff
@@ -28,6 +28,10 @@ type Config struct {
 	// ReadOnly specifies a type of validation.
 	// If ReadOnly set 'true' validator will run all checks that don't make any modifications.
 	ReadOnly bool
+
+	// ReadLimit specifies whether to limit the number of account read
+	// If ReadLimit set 'true', check for account and entitlement read will only read 8 accounts
+	ReadLimit bool
```

Comment:
I think we want to make this an integer value that is configurable.

Reply:
Done